python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -10,11 +10,14 @@
 from ...status.models.mine_status_xref import MineStatusXref
 from ..models.mine_identity import MineIdentity
 from ..models.mine_detail import MineDetail
+from ..models.mine_tenure_type_code import MineTenureTypeCode
 from ..models.mine_type import MineType
 from ..models.mineral_tenure_xref import MineralTenureXref
 from ....permits.permit.models.permit import Permit
+from ...region.models.region import MineRegionCode
```

Comment:
@james-hollinger @Jsyro is this an example of an incorrect way of using a method directly? I've only overheard some discussions about sending internal API requests, but I haven't been in those conversations. Is that required here?

Reply:
@NWCalvank Good eyes, this is fine as they are under the same module '/mines/mine' and '/mines/region' (aka, they will always be in the same microservice). if this was fetching a document or a party, we want to use REST calls as those modules may be separated into separate microservices in the future.  This is good as is.